include/wx/generic/listctrl.h

```diff
@@ -216,6 +216,9 @@ class WXDLLIMPEXP_CORE wxGenericListCtrl: public wxNavigationEnabled<wxListCtrlB
     // return the text for the given column of the given item
     virtual wxString OnGetItemText(long item, long column) const;
 
+    // return whether the given item is checked
+    virtual bool OnGetItemIsChecked(long item) const;
```

Comment:
This is consistent with the other methods, but seems a bit awkward to me. I wonder if `OnItemIsChecked()` could be better while still retaining a degree of consistency?

But I'm leaving the final choice of the name to you here, this is just a suggestion.